### DISCUSSION TITLE
Incorrect status information

### DISCUSSION THREAD

**Discussion by molomby** [forwardemail/forwardemail.net#206] [General]
Forward email had an outage about 12 hrs ago, effecting the website and the API, for at least 30 mins. At the time, several issues were created in the status repo: [#100](https://github.com/forwardemail/status.forwardemail.net/issues/100), [#101](https://github.com/forwardemail/status.forwardemail.net/issues/101) and [#102](https://github.com/forwardemail/status.forwardemail.net/issues/102), all now deleted. Issue [#102](https://github.com/forwardemail/status.forwardemail.net/issues/102) described work being done on the IPv6 stack in what sounded like a scheduled upgrade/testing.

**My concern isn't that downtime occurred.** I host systems too and 100% uptime is an impossible dream – outages happen and scheduled downtime for maintenance is often the only practical solution to a problem. I find it a little concerning though that downtime like this isn't correctly shown on the status page.

At the time the downtime occurred, the status repo correctly reported that `api.forwardemail.net` was offline but incorrectly reported the main `forwardemail.net` website as online. This persisted for at least 30 mins. And then today, it seems the outage has been excluded entirely from the record – both the API and public sites show 100% uptime for the last 24 hrs, with no record of the downtime at all.

I don't expect any service to always be online but I do expect a level of transparency. Whats the point in having a public status page if it can't be trusted?

**Comment by molomby** [comment]
I see [status.forwardemail.net](https://status.forwardemail.net/) does report some downtime on the API in the last 24 hrs – 99.59% up, equivalent to ~6 mins of downtime. This still significantly understates the downtime I personally observed but possibly there was a geographic component to the outage? I also note my connection here doesn't support IPv6 (which, apparently, was being tested), so it's possible the service was available for others while I experienced a 30+ minute period of unavailability...?

Regardless, the public site outage I saw for the same period isn't recorded anywhere. And _surely_ incident reporting/follow up (such as the deleted GitHub issues) should remain public, if not only so users like me aren't forced to speculate on the duration, severity, geography or existence (!) of issues as I'm doing now.

**Reply by titanism** [reply]
Hi there!   The status uptime is accurate, we did experience outage with IPv4/IPv6 for intermittent period during maintenance window last evening.  This is all public and available in commit history and from the graphs.

**Reply by titanism** [reply]
Also we will definitely try to improve on maintenance window and notices to customers!  Thank you @molomby

**Reply by titanism** [reply]
@molomby I think that the automatic deletion is built-in to the status uptime tool we're using; we can definitely take a look to try to disable that feature.  We use https://github.com/upptime/upptime under the hood.
